stage_chunk/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gcr.io/dataflow-templates-base/python3-template-launcher-base:latest
+
+# Copy your pipeline code inside the container
+WORKDIR /dataflow/template
+COPY stage_chunk_beam_job.py .
+
+# Install dependencies
+RUN pip install --no-cache-dir google-cloud-storage
```

Comment:
Do you also need dependencies from `requirements.txt` here?

Reply:
The `requirements.txt` is used automatically when deploying the function because of the `--source=.` command line switch in `deploy-function.sh`.

The Dataflow job only has the single extra dependency besides those included in the base Docker image, so it is just installed manually.

This could be made clearer by having a separate requirements file for the Dataflow job from the function, e.g., but it did not seem necessary just for installing the one package.